dashboard/v1.1/src/pages/Monitoring/RouteDetails.tsx

```diff
@@ -115,23 +122,49 @@ const RouteDetailsComponent: FC<RouteDetailsProps> = ({
   const data = format(routesChains);
   return (
     <>
-      <span
-        style={{ display: 'flex', alignItems: 'center' }}
-        onClick={() => showDetails(false, routesChains)}
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'space-between',
+          alignItems: 'center',
+          position: 'relative'
+        }}
       >
-        <ArrowBackIcon color="primary" fontSize="large" />
-        <span
-          style={{
-            fontSize: '1rem',
-            fontWeight: 'bold',
-            padding: '0 0.4rem',
-            display: 'flex',
-            alignItems: 'center'
-          }}
-        >
-          {truncate(data.name, 70)}
+        <span style={{ display: 'flex', position: 'relative', width: '70%' }}>
+          <ArrowBackIcon
+            color="primary"
+            fontSize="large"
+            style={{ cursor: 'pointer ' }}
+            onClick={() => showDetails(false, routesChains)}
+          />
+          <h3
+            style={{
+              fontSize: '1rem',
+              fontWeight: 'bold',
+              padding: '0.4rem 0.4rem 0 0.4rem',
+              margin: 0,
+              maxWidth: '90%',
+              // width: '200px',
+              overflow: 'hidden',
+              textOverflow: 'ellipsis',
+              whiteSpace: 'nowrap'
+            }}
+          >
+            {/* {truncate(data.name, 70)} */}
```

Comment:
Is this PR ready for review? Please request reviews once it is and remove comments before doing that

Reply:
Thanks for the reply @aquibbaig, will do the prescribed changes and request for review.